src/database/models/user.js

```diff
@@ -1,10 +1,17 @@
 
 module.exports = (sequelize, DataTypes) => {
-  const User = sequelize.define('User', {
-    email: DataTypes.STRING
+  const user = sequelize.define('user', {
+    firstname: DataTypes.STRING,
+    lastname: DataTypes.STRING,
+    email: DataTypes.STRING,
+    password: DataTypes.STRING,
+    gender: DataTypes.STRING,
+    address: DataTypes.STRING,
+    role: DataTypes.STRING,
+    isVerified: DataTypes.BOOLEAN
   }, {});
-  User.associate = (models) => {
+  user.associate = (models) => {
```

Comment:
could you remove this line too? We saw that we don't need it so far. I suggest this because it decreasing functions coverage